server/python/src/routers/movies.py

```diff
@@ -1,5 +1,5 @@
 from fastapi import APIRouter, Query, Path, Body
-from src.database.mongo_client import db, get_collection
+from src.database.mongo_client import db, get_collection, voyage_ai_available
```

Comment:
Looks like this is an unused import.

```suggestion
from src.database.mongo_client import get_collection, voyage_ai_available
```